core/src/main/scala/akka/persistence/jdbc/schema/scaladsl/SchemaUtils.scala

```diff
@@ -0,0 +1,131 @@
+    val slickDb: SlickDatabase = SlickExtension(systemProvider).database(journalConfig)
+
+    val (file, separator) =
+      slickDb.profile match {
+        case PostgresProfile  => ("schema/postgres/postgres-schema.sql", ";")
+        case MySQLProfile     => ("schema/mysql/mysql-schema.sql", ";")
+        case OracleProfile    => ("schema/oracle/oracle-schema.sql", "/")
+        case SQLServerProfile => ("schema/sqlserver/sqlserver-schema.sql", ";")
+        case H2Profile        => ("schema/h2/h2-schema.sql", ";")
+      }
+
+    val blockingEC = actorSystem.dispatchers.lookup(Dispatchers.DefaultBlockingDispatcherId)
+    applySchema(slickDb.database, file, separator, blockingEC)
+  }
+
+  /**
+   * For testing purposes the journal and snapshot tables can be created programmatically.
+   *
+   * This method can be used to load alternative schema scripts. The passed schema argument must be a file accessible from the classpath.
+   *
+   * By default it will use the database settings found under `jdbc-journal`, or `akka-persistence-jdbc.shared-databases` if configured so.
+   * See https://doc.akka.io/docs/akka-persistence-jdbc/current/index.html#sharing-the-database-connection-pool-between-the-journals for details.
+   *
+   * To use the snapshot configuration, pass `jdbc-snapshot-store` to `configKey`.
+   *
+   * @param schema the DDL script file. Must be accessible from the classpath.
+   * @param separator used to separate the different DDL statement. Defaults to ';' for most databases.
+   * @param configKey the configuration key to use, default to `jdbc-journal`
+   * @param systemProvider a ClassicActorSystemProvider
+   * @return
+   */
+  def applySchema(schema: String, separator: String = ";", configKey: String = "jdbc-journal")(
+      implicit systemProvider: ClassicActorSystemProvider): Future[Done] = {
+
+    val actorSystem = systemProvider.classicSystem
+    val journalConfig = actorSystem.settings.config.getConfig(configKey)
+    val slickDb: SlickDatabase = SlickExtension(actorSystem).database(journalConfig)
+    val db: Database = slickDb.database
+
+    val blockingEC = actorSystem.dispatchers.lookup(Dispatchers.DefaultBlockingDispatcherId)
+    applySchema(db, schema, separator, blockingEC)
+  }
+
+  private def applySchema(
+      db: Database,
+      schema: String,
+      separator: String,
+      blockingEC: ExecutionContext): Future[Done] = {
+    Future {
+      for {
+        schema <- Option(fromClasspathAsString(schema))
+        ddl <- for {
+          trimmedLine <- schema.split(separator).map(_.trim)
+          if trimmedLine.nonEmpty
+        } yield trimmedLine
+      } withStatement(db) { stmt =>
+        logger.debug(s"applying DDL: $ddl")
+        try stmt.executeUpdate(ddl)
+        catch {
+          case t: SQLSyntaxErrorException if shouldSuppressKnownError(t.getMessage) =>
+          case t: java.sql.SQLException if shouldSuppressKnownError(t.getMessage)   =>
```

Comment:
Not worth logging at debug because we are so sure they are never interesting?

Reply:
Yes, will do it